foodji_ui/lib/models/measurement_model.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:faker/faker.dart';
+import 'package:foodji_ui/models/unit_type_enum.dart';
+
+class MeasurementModel {
+  String alternativeText;
+  String unitType;
+  num value; // Must be a num, which is parent to double and int, as received data can be interpreted as either by json convert
+
+  MeasurementModel(
+      {required this.alternativeText,
+      required this.unitType,
+      required this.value});
+
+  factory MeasurementModel.fromJson(Map<String, dynamic> json) {
+    return MeasurementModel(
+        alternativeText: json['alternativeText'],
+        value: json['value'],
+        unitType: json['unitType']);
+  }
+
+  static MeasurementModel getSample() {
+    var faker = Faker();
+
+    return MeasurementModel(
+        alternativeText: faker.food.cuisine(),
+        value: faker.randomGenerator.decimal(),
+        unitType: faker.randomGenerator.element(UnitType.values).name);
+  }
+
+  static List<MeasurementModel> getSamples(int amount) {
+    var samples = <MeasurementModel>[];
+
+    for (var i = 0; i < amount; i++) {
```

Comment:
les accolades sur la même ligne ou pas sur la même ligne? constancy is key

Reply:
J'ai vérifié ce fichier en entier ainsi que d'autres fichiers .dart pour confirmer ; l'autoformattage .dart positionne les brackets en fin de ligne à l'ouverture, et sur la ligne suivante en fermeture. Ce fichier même, ni les autres qui ont étés vérifiées, ne présentent aucune inconsistence.